test/Libraries/RevitIntegrationTests/RevitWatch3DViewModelTests.cs

```diff
@@ -11,7 +11,7 @@ namespace RevitSystemTests
     public class RevitWatch3DViewModelTests : RevitSystemTestBase
     {
         [SetUp]
-        public void Setup()
+        public override void Setup()
         {
```

Comment:
warning about 'hiding' and it turned out in this case "Setup" was a virtual, so added override...